arch/arm/boot/dts/overlays/Makefile

```diff
@@ -130,6 +130,8 @@ dtbo-$(CONFIG_ARCH_BCM2835) += \
 	justboom-dac.dtbo \
 	justboom-digi.dtbo \
 	ltc294x.dtbo \
+	lt6911uxc-overlay.dtbo \
```

Comment:
Don't include the `-overlay` here - there is a make rule to create `*.dtbo` from `*-overlay.dts`.

Also, both new entries belong before `ltc294x.dtbo'.

Reply:
resolved in last commit